runtime/dolphin/src/lib.rs

```diff
@@ -561,6 +562,24 @@ impl pallet_treasury::Config for Runtime {
     type SpendOrigin = NeverEnsureOrigin<Balance>;
 }
 
+impl pallet_aura_style_filter::Config for Runtime {
+    /// Nimbus filter pipeline (final) step 3:
+    /// Choose 1 collator from PotentialAuthors as eligible
+    /// for each slot in round-robin fashion
+    type PotentialAuthors = CollatorSelection;
+}
+
+impl pallet_author_inherent::Config for Runtime {
+    // We start a new slot each time we see a new relay block.
+    type SlotBeacon = cumulus_pallet_parachain_system::RelaychainBlockNumberProvider<Self>;
+    type AccountLookup = CollatorSelection;
+    type EventHandler = ();
+    type WeightInfo = (); // TODO: Add benchmarked weights
```

Comment:
can you add this the benchmark workflows and maybe add a fake weights file, and we'll replace it with a real one in the release PR.

Reply:
done